lighthouse-core/gather/gatherers/source-maps.js

```diff
@@ -28,7 +28,7 @@ class SourceMaps extends Gatherer {
    */
   async fetchSourceMap(driver, sourceMapUrl) {
     /** @type {string} */
-    const sourceMapJson = await driver.fetcher.fetchResource(sourceMapUrl, {timeout: 1500});
+    const sourceMapJson = await driver.fetcher.fetchResource(sourceMapUrl, {timeout: 10000});
```

Comment:
what's up with this timeout bump?

Reply:
I was hitting the 1500 timeout pretty consistently on preactjs.com

Reply:
> I was hitting the 1500 timeout pretty consistently on preactjs.com

10s might be too long to block on a single gatherer, though :/

Was this before or after you added `_fetchResourceOverProtocol`, though? Because if it was after and with the current timeout approach, that would mean just `readIOStream` is often taking longer than 1.5s?

Reply:
It was after. The iframe method completed within 1.5s.

Reply:
hmm, I think something else is going on with preactjs.

For the iframe method, I can now repro the file download behavior from #12064 using the CLI for the file `bundle.dd34e.esm.js.map`.  That looks like it never resolves the `requestInterceptionPromise` and so it always times out no matter how long I set the timeout.

For the `Network.loadNetworkResource` approach it never times out for me. Instrumenting the function and running it several times, I get a mean time of 176ms for the `Network.loadNetworkResource` call and 14ms for `driver.readIOStream()`, way below the timeout.

Reply:
Yeah this appears to be an issue with preactjs.com. I'm successfully fetching source maps on other sites. Leaving this timeout as 1500 should be good.